Added voice over accessibility support

JamesJSY opened the pull request:
The previous one has some conflicts with the master branch and I cannot update it, even with -f flag. So I created a new pr.

ppatriotis commented on Pod/Classes/UI/CardView.xib:
```diff
@@ -48,6 +50,7 @@
                                 <textField opaque="NO" clipsSubviews="YES" contentMode="scaleToFill" contentHorizontalAlignment="left" contentVerticalAlignment="center" placeholder="YY" minimumFontSize="17" translatesAutoresizingMaskIntoConstraints="NO" id="CgE-Em-qHJ" customClass="YearInputTextField" customModule="Caishen" customModuleProvider="target">
                                     <rect key="frame" x="272" y="0.0" width="37" height="596"/>
                                     <color key="backgroundColor" white="0.0" alpha="0.0" colorSpace="calibratedWhite"/>
+                                    <accessibility key="accessibilityConfiguration" label="Year for Expiration Date"/>
```
@Haud shouldn't this be "Expiration Year"

Haud replied:
Yea, that would make more sense.

ppatriotis commented on Pod/Classes/UI/CardView.xib:
```diff
@@ -39,6 +40,7 @@
                                 <textField opaque="NO" clipsSubviews="YES" contentMode="scaleToFill" contentHorizontalAlignment="right" contentVerticalAlignment="center" placeholder="MM" textAlignment="right" minimumFontSize="17" translatesAutoresizingMaskIntoConstraints="NO" id="5cX-Z5-afy" customClass="MonthInputTextField" customModule="Caishen" customModuleProvider="target">
                                     <rect key="frame" x="229" y="0.0" width="37" height="596"/>
                                     <color key="backgroundColor" white="0.0" alpha="0.0" colorSpace="calibratedWhite"/>
+                                    <accessibility key="accessibilityConfiguration" label="Month for Expiration Date"/>
```
@Haud shouldn't this be "Expiration Month"

Haud replied:
Yes

ppatriotis commented on Pod/Classes/UI/CardView.xib:
```diff
@@ -63,6 +66,7 @@
                                 <textField opaque="NO" clipsSubviews="YES" contentMode="scaleToFill" contentHorizontalAlignment="left" contentVerticalAlignment="center" placeholder="CVC" textAlignment="right" minimumFontSize="17" translatesAutoresizingMaskIntoConstraints="NO" id="HOM-PC-B5m" customClass="CVCInputTextField" customModule="Caishen" customModuleProvider="target">
                                     <rect key="frame" x="309" y="0.0" width="221" height="596"/>
                                     <color key="backgroundColor" white="0.0" alpha="0.0" colorSpace="calibratedWhite"/>
+                                    <accessibility key="accessibilityConfiguration" label="CVC for Card"/>
```
@Haud shouldn't this be "Card Verification Code"?

Haud replied:
Yup

ppatriotis commented:
How would translations to other languages work?

JamesJSY commented:
Thanks for pointing out @ppatriotis. Fixed accessibility label text and also added localization support for English, Chinese and German. Huge thanks to @DannyVancura :100: 

Users can either update the property accessibilityLabel on each textfield to fit their needs for voice over accessibility support. Or make a new pr to add new localizable.strings file to support their language.

DannyVancura commented:
To consider this full voice over accessibility support we should think about audible notifications that the provided card number or expiration date were invalid (i.e. make color changes audible).

JamesJSY commented:
Added audible notifications that the provided card number was invalid and the provided expiration date had expired. @ppatriotis @Haud 

Enormous thanks to @DannyVancura

Haud commented on Caishen.podspec:
```diff
@@ -29,5 +29,5 @@ Pod::Spec.new do |s|
 
   s.source_files = 'Pod/Classes/**/*'
-  s.resources = 'Pod/Assets/*'
+  s.resources = ['Pod/Assets/*', 'Pod/Resources/*']
```
Why did we need a separate folder for this / what's the difference?

DannyVancura replied:
The difference is more or less the kind of content. `Assets` for image resources and the like, whereas `Resources` in this case are mostly .strings files for the localization.

Haud commented on Section 508.md:
```diff
@@ -0,0 +1,54 @@
+# Section 508
```
Why was this added? Does anything link to this?

DannyVancura replied:
This just describes the compliance to governmental standards w.r.t. accessibility for visually impaired users, which is added in this PR.

Haud commented:
Where did the translations come from? Are they from native speakers?

DannyVancura commented:
Yes. German translations from my end, Chinese from @JamesJSY. Both native speakers.
